loadfile/loadfile_test.go

```diff
@@ -21,21 +21,20 @@ func Test_LoadContext(t *testing.T) {
 	assert.NoError(t, os.MkdirAll(testdir, os.ModePerm))
 
 	// create a directory
-	dirname := "mydir"
-	dirpath := filepath.Join(testdir, dirname)
-	assert.NoError(t, os.MkdirAll(dirpath, os.ModePerm))
+	dirPrefix := "mydir"
+	dirPath, err := os.MkdirTemp(testdir, dirPrefix)
```

Comment:
I almost hate to even say this, but since `dirPrefix` is only used once, you could just inline the literal and get rid of the variable. Same for `filenamePrefix`, below.